### PR TITLE
Feature/login button

### LINKED ISSUE #25: Login Button
This login button needs to use the google API

https://www.npmjs.com/package/react-google-login

Restrict to cedarville.edu emails.

Once you login and get your JWT check it against the backend. If fail, go to register page

An example: https://github.com/CreativeSolutionsGroup/smart-events-admin/blob/master/src/components/Login.js

### PR DESCRIPTION
**Pull request by ashlyndevries**
Close #25

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,9 +14,11 @@
     "@fontsource/roboto": "^4.5.7",
     "@mui/icons-material": "^5.8.4",
     "@mui/material": "^5.9.2",
+    "@react-oauth/google": "^0.2.8",
     "axios": "^0.27.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-google-login": "^5.2.2",
```
Comment (SpBills) [inline package.json:21]: Remove this dependency: `react-google-login`. (just remove the line)

File: src/main.tsx [hunk 2]
```diff
@@ -2,17 +2,19 @@ import React from 'react'
...
-import CheckIn from './components/CheckInTable'
 import './index.css'
 import '@fontsource/roboto/300.css';
 import '@fontsource/roboto/400.css';
 import '@fontsource/roboto/500.css';
 import '@fontsource/roboto/700.css';
-import { Typography } from '@mui/material'
+import { GoogleOAuthProvider } from '@react-oauth/google'
+
+const clientId = '8990298927-tftcl4kcfe52ikmu3quthlo7v3qvognu.apps.googleusercontent.com';
```
Comment (SpBills) [inline src/main.tsx:12]: No action needed, but I don't like clientId being static here. We'll probably introduce an env variable for it eventually.

**Review by SpBills — CHANGES_REQUESTED** [review]
Just remove one line! Great job with this @SebastianNeiswanger @ashlyndevries !!!

File: src/main.tsx (hunk 2 above)
Comment (SebastianNeiswanger) [reply src/main.tsx:12]: That makes sense. This one was just for testing and can be changed to an env once we are ready for production

File: package.json (hunk 1 above)
Comment (SebastianNeiswanger) [reply package.json]: fixed with new commit

**Review by SpBills — APPROVED** [review]
LGTM. Thank you guys, this is a huge step.

**Pull request merged**